android/src/main/java/com/bradbumbalough/RCTTwilioChat/RCTTwilioChatChannels.java

```diff
@@ -556,6 +556,11 @@ public void onSuccess(Integer count) {
         });
     }
 
+    @ReactMethod
+    public void shutdown() {
+        channelListeners.clear();
```

Comment:
I also attempted to keep a list of all Channels created, loop through them here and call `channel.removeAllListeners()`. However I was getting errors saying that the channel object had already been disposed. So for now, I'm just keeping it to clearing the HashMap.